apps/emqx_bridge_greptimedb/src/emqx_bridge_greptimedb_connector.erl

```diff
@@ -164,6 +214,14 @@ on_get_status(_InstId, #{client := Client}) ->
             disconnected
     end.
 
+transform_bridge_v1_config_to_connector_config(BridgeV1Config) ->
```

Comment:
Nit: maybe move this to the action info module?

Reply:
I implemented this function following influxdb connector but then realized that it was needed only for some influxdb tests but forgot to remove it in greptimedb bridge... Now I've removed it, thanks. Note: greptimedb action module already has a similar function.